src/pages/home/components/Sidebar/components/SocialMediaForm/SocialMediaForm.tsx

```diff
@@ -0,0 +1,84 @@
+import { ReactNode } from 'react';
+
+import Button from '~components/Button/Button';
+
+import scss from './SocialMediaList.module.scss';
+
+import { SocialMediaFormProps } from './SocialMediaForm.types';
+
+function SocialMediaForm({
+  onHandleToggleModal,
+}: SocialMediaFormProps): ReactNode {
+  return (
+    <ul className={scss.selection}>
+      <li className={scss.socialItem}>
+        <button type="button">
+          <img alt="facebook" src="./social-github.svg" />
+          <p>Facebook</p>
+          <Button
+            className={scss.button}
+            onClick={onHandleToggleModal}
+            variant="outlined"
+          >
+            Connect Account
+          </Button>
+          <Button
+            className={scss.button}
+            onClick={onHandleToggleModal}
+            variant="container"
+          >
+            Connect Group
+          </Button>
+        </button>
+      </li>
+      <li className={scss.socialItem}>
+        <button type="button">
+          <img alt="linkedin" src="./social-github.svg" />
+          <p>linkedin</p>
+          <Button
+            className={scss.button}
+            onClick={onHandleToggleModal}
+            variant="outlined"
+          >
+            Connect Account
+          </Button>
+          <Button
+            className={scss.button}
+            onClick={onHandleToggleModal}
+            variant="container"
+          >
+            Connect Group
+          </Button>
+        </button>
+      </li>
+      <li className={scss.socialItem}>
+        <button type="button">
+          <img alt="twitter" src="./social-github.svg" />
+          <p>twitter</p>
```

Comment:
Isso não é um `p` é um `span`

Tambem colocar o nome da rede com a primeira letra maiuscula em todos os casos

Reply:
reafactor tag p for span and add class